### DISCUSSION TITLE
Widgets for multiple habits on homescreen

### DISCUSSION THREAD

**Discussion by lotje1990** [iSoron/uhabits#1073] [Feature Requests] [answered]
I wish i could have more widgets of the same type on my homescreen. Like for 3 habits a shortcut to check it off. Now i can only have 1 check-it widget, while i have multiple habits for 1 day...

**Comment by hiqua** [comment]
I didn't fully understand what you're talking about, but there are stacked widgets in v2 which might address your needs: basically you can put checkmarks that have several habits, you have to swipe up to view the ones behind.

**Comment by iSoron** [comment, marked as answer]
@lotje1990 It should already be possible to add multiple widgets of the same type in your home screen. See screenshots in Google Play. If your phone does not allow you to add multiple checkmark widgets, for example, then this is a limitation of your phone, or more specifically, of your phone's launcher. You can solve this issue by installing a 3rd party launcher, such as [Nova Launcher](https://novalauncher.com/).

**Comment by VA-MMXXII** [comment]
Thought a bit tedious to set up, another idea is to use Popup Widget 3 (https://play.google.com/store/apps/details?id=com.ss.popupWidget&hl=en_US&gl=US) which allows you to make groups of widgets pop up by clicking on a single icon. 

This keeps your home screen cleaner and more organized and you can duplicate widget settings for a more standardized look.

You can also backup your new widget shortcuts and if your home screen gets disturbed by an Android update, it's easy to recreate them (although still requires some manual placement). I've used it successfully with the native Pixel launcher, MS Launcher, and Nova Launcher. The latter two are recommended as they also allow you to back up your entire configuration.
